engine/gamesys/src/gamesys/components/comp_collision_object.cpp

```diff
@@ -343,6 +343,9 @@ namespace dmGameSystem
                     {
                         uint32_t cell_x = cell->m_X - tile_grid_resource->m_MinCellX;
                         uint32_t cell_y = cell->m_Y - tile_grid_resource->m_MinCellY;
+                        flags.m_FlipHorizontal = cell->m_HFlip;
+                        flags.m_FlipVertical = cell->m_VFlip;
+                        flags.m_Rotate90 = cell->m_Rotate90;
```

Comment:
We never fill the flags. It seems like this was the issue for years and I missed it as well last time I worked on it.